src/scripts/db_tool.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# nmdc_schema/src/scripts/db_tool.py
+"""
+db_tool.py: Provide command-line tools for interacting with the NMDC database.
```

Comment:
let's think of a more explicit name. something that reflects that it is retrieving related records from the database

Reply:
nmdc_database_tools